astropy/convolution/tests/test_convolve_fft.py

```diff
@@ -36,7 +36,13 @@
                                  NANTREATMENT_OPTIONS,
                                  (True, False),
                                  ))
-
+option_names_preserve_nan = \
```

Comment:
no need for `\`, please start listing the items in this line, that case it can fit easily in 2 rather than 3 lines and stays just as readable

Reply:
:+1: to using `(`'s natural line continuation over the explicit continuation from `\`.